### PR TITLE
Upload coverage to coveralls

### LINKED ISSUE #311: Setup code coverage collection

### PR DESCRIPTION
**Pull request by collierrgbsitisfise**
Fixes #311

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +40,7 @@
   "devDependencies": {
     "chalk": "2.4.2",
     "chromedriver": "76.0.1",
+    "coveralls": "^3.0.6",
```
Comment (marco-c) [inline package.json]: Could you fix the version, like we do for the other devDependencies?

File: package.json [hunk 2]
```diff
@@ -47,6 +49,7 @@
     "istanbul": "0.4.5",
     "mkdirp": "0.5.1",
     "mocha": "6.2.0",
+    "mocha-lcov-reporter": "^1.3.0",
```
Comment (marco-c) [inline package.json]: Same here

File: package.json (hunk 1 above)
Comment (collierrgbsitisfise) [reply package.json]: yes, sure

File: package.json (hunk 2 above)
Comment (collierrgbsitisfise) [reply package.json]: yes, sure

**Comment by collierrgbsitisfise** [comment]
@marco-c , hi just did an fix according CR, just curious - what is a reason lock versions of 
 devDep(only) ?

**Comment by marco-c** [comment]
> @marco-c , hi just did an fix according CR, just curious - what is a reason to lock versions of
devDep(only) ?

I generally prefer locking dependencies, to make things reproducible. Since this is a library, the normal dependencies can't be locked (or all the projects depending on the library will inherit the locked dependencies), but we can lock the devDependencies at least.

**Comment by marco-c** [comment]
Thanks!

**Pull request merged**